config/schedule.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 env :PATH, ENV['PATH']
 
-# Every minute, check for departure inaccuracies
-every 60.seconds do
+# Every minute after 5am, check for departure inaccuracies
+# raw cron asterisk order: minute, hour, day of month, month, day of week
+every '*/1 6-23 * * *' do
```

Comment:
I'm pretty sure you can write `*` instead of `*/1`

Two other things:
*  Comment says after 5:00am, but this starts running every minute after 6:00am, right?
*  We also have service on the road until 3:00am.  I wonder of the hours should be `0-3,5-23`?